Formula/julius.rb

```diff
@@ -15,6 +15,10 @@ class Julius < Formula
 
   depends_on "libsndfile"
 
+  uses_from_macos "zlib"
+
+  patch :DATA
```

Comment:
Is this upstreamed?

Reply:
I upstreamed this fix and linked to the pull request in a comment.